lib/logstash/filters/pilar.rb

```diff
@@ -38,9 +38,14 @@ class Pilar < LogStash::Filters::Base
       # default will be 'message', matching the default format in the `logformat` variable
       config :content_specifier, validate: :string, required: false, default: 'message'
 
+      # The regex is UNKNOWN (TODO: Follow Up With Weiyi About Purpose)
+      config :regexes, validate: :array, required: true, default: ['(\d+\.){3}\d+']
```

Comment:
Lets add some comments to this now that we know what this is talking about. Also a more descriptive name might be better?